backend/src/main/kotlin/br/alisson/poc/model/Imc.java

```diff
@@ -0,0 +1,53 @@
+package br.alisson.poc.model;
+
+import java.time.Instant;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+@Entity
+@Table(name = "tb_imc")
+public class Imc {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE)
+	@Column(name = "co_imc", nullable = false, unique = false)
```

Comment:
```suggestion
	@Column(name = "co_imc", nullable = false, unique = true)
```
Um id não seria único ?